### ISSUE TITLE
Scatter3DPlot raises a null pointer

### ISSUE THREAD

**Issue by ajnebro** [jtablesaw/tablesaw#721]
Hi.
  I'm trying to use a 3D scatter plot with this code:

```java 
    DoubleColumn xData = DoubleColumn.create("x", new double[]{2, 2, 1});
    DoubleColumn yData = DoubleColumn.create("y", new double[]{1, 2, 3});
    DoubleColumn zData = DoubleColumn.create("z", new double[]{1, 4, 1});

    Table data = Table.create().addColumns(xData, yData, zData) ;

    Plot.show(Scatter3DPlot.create("3D plot", data, "x", "y", "z"));
```

but I'm getting the following exception:
```
Exception in thread "main" java.lang.NullPointerException
	at tech.tablesaw.plotly.traces.Scatter3DTrace.<init>(Scatter3DTrace.java:46)
	at tech.tablesaw.plotly.traces.Scatter3DTrace.<init>(Scatter3DTrace.java:16)
	at tech.tablesaw.plotly.traces.Scatter3DTrace$Scatter3DBuilder.build(Scatter3DTrace.java:149)
	at tech.tablesaw.plotly.api.Scatter3DPlot.create(Scatter3DPlot.java:47)
```

I'm executing the code in both Windows 10 and macos (Catalina) with Java 8 (version "1.8.0_221") and version 0.36.0 of tablesaw-core and tablesaw-jsplot.

**Comment by lwhite1** [comment]
fixed in PR #764

**Comment by lwhite1** [comment]
fixed
